scraper.ts

```diff
@@ -1,21 +1,20 @@
-// Reads and parses anchor data files from bikeshed-data repository and writes:
+// Reads and parses anchor data files from webref repository and writes:
 // - xref.json containing parsed and formatted data by term
 // - specs.json having data by spec shortname
 // - specmap.json having spec details
 
 import { promises as fs, existsSync } from 'fs';
 import { resolve as resolvePath, join as joinPath } from 'path';
 import { spawn } from 'child_process';
-import Trie from 'compact-prefix-tree/cjs';
```

Comment:
Lets also `npm un compact-prefix-tree --save` and `rm @types/compact-prefix-tree.d.ts`

Reply:
We're doing pair programming here :smile: 
Lets do live coding someday!